internal/cli/output_opts.go

```diff
@@ -93,10 +95,16 @@ func (opts *OutputOpts) Print(v interface{}) error {
 	if opts.ConfigOutput() == jsonFormat {
 		return jsonwriter.Print(opts.ConfigWriter(), v)
 	}
-	t, err := opts.parseTemplate()
+
+	outputFormat, t, err := opts.parseTemplate()
 	if err != nil {
 		return err
 	}
+
+	if outputFormat == jsonPath && t != "" {
+		return jsonpathwriter.Print(opts.ConfigWriter(), t, v)
+	}
```

Comment:
[q] if `t` is empty and json path is not should we return an error? what would happen in that case?

Reply:
it returns something weird. I will add a check for an empty path in the writer
```
./bin/mongocli atlas privateendpoint aws get 60100269480a253a4044da93 -o json-path=
&{60100269480a253a4044da93   com.amazonaws.vpce.eu-central-1.vpce-svc-0d2136ac750f0244b  [] []   WAITING_FOR_USER}
```